app/tests/test.py

```diff
@@ -3,11 +3,15 @@
 import time
 import os
 
+is_travis = 'TRAVIS' in os.environ	

```

Comment:
Remove trailing whitespaces. Multiple places in this file

Reply:
@srungta can you please specify line number you are talking about 😄

Reply:
@srungta I checked the whole file and it's not of autopep8 standard. If we go through autopep8standard than i have to add more space. Please guide me if i miss something

Reply:
@vaibhavsingh97 There is some whitespace after `os.environ`, can you please remove it?
Thanks

Reply:
The white space is giving you the "Not so good... This pull request quality could be better." in codacy. 
https://www.codacy.com/app/fossasia/badgeyay/pullRequest?prid=935773

Reply:
Ohh! I got it